### DISCUSSION TITLE
"Stagger" plotting

### DISCUSSION THREAD

**Discussion by farchord** [madMAx43v3r/chia-plotter#571] [Ideas]
So, we have a new Threadripper machine, and well, the name is well earned, it rips through plots. We're planning on chucking hard drives (We're using WD Elements and we're fully aware of the issues with the SATA pins), but in the mean time we're using a USB hard drive as a final plot.

As you may guess, right now, we're running an issue where it almost takes as long to copy a plot to the final destination than it takes to create the plot. 

Nice problem to have, but my question would be, how does the plotter handle the situation where there's 2+ file copies in progress, and it's starting to get too many?

Is there a way for the plotter to "wait" once it gets to a certain amount of simultaneous file copies (i.e. 2), or maybe have it spread the final destinations to more than one folder? (i.e. plot #1 would go to final destination #1, plot #2 to destination #2, plot #3 to #1, and so on)

Thanks!

**Comment by SebMoore** [comment]
> So, we have a new Threadripper machine, and well, the name is well earned, it rips through plots. We're planning on chucking hard drives (We're using WD Elements and we're fully aware of the issues with the SATA pins), but in the mean time we're using a USB hard drive as a final plot.
> 
> As you may guess, right now, we're running an issue where it almost takes as long to copy a plot to the final destination than it takes to create the plot.
> 
> Nice problem to have, but my question would be, how does the plotter handle the situation where there's 2+ file copies in progress, and it's starting to get too many?
> 
> Is there a way for the plotter to "wait" once it gets to a certain amount of simultaneous file copies (i.e. 2), or maybe have it spread the final destinations to more than one folder? (i.e. plot #1 would go to final destination #1, plot #2 to destination #2, plot #3 to #1, and so on)
> 
> Thanks!

There is currently no way to spread the plots. The plotter will pause plotting, however, if a plot finishes and there is one still copying.

**Reply by farchord** [reply]
Ah I did not know that! That pretty much answers my question and worry, thanks!

**Reply by SebMoore** [reply]
No worries!
